Source/WebKit/UIProcess/Extensions/Cocoa/_WKWebExtensionStorageSQLiteStore.mm

```diff
@@ -71,14 +71,34 @@ - (instancetype)initWithUniqueIdentifier:(NSString *)uniqueIdentifier storageTyp
     return self;
 }
 
+- (void)getAllKeys:(void (^)(NSArray<NSString *> *keys, NSString *errorMessage))completionHandler
+{
+    auto weakSelf = WeakObjCPtr<_WKWebExtensionStorageSQLiteStore> { self };
```

Comment:
This syntax should work now, I fixed `WeakObjCPtr` recently to support it.

```suggestion
    auto weakSelf = WeakObjCPtr { self };
```

Reply:
Sounds good. I've gone ahead and updated all the instances across the codebase I could find that could use this. What's the policy on cleanup patches? ...should I make a new PR for that or add the new commit to this one?

Reply:
If it is just in this file, you can do it in this PR. Or if it is broader than just extension files, just file a new bug and post a clean up PR.